src/pages/sdk/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { theme } from '@apify-packages/ui-library';
+import Layout from '@theme/Layout';
+import React from 'react';
+import styled from 'styled-components';
+
+import Hero from '../../components/Hero/Hero';
+import SdkSection from '../../components/SdkSection/SdkSection';
+import Section from '../../components/Section/Section';
+import UiLibraryWrapper from '../../components/UiLibraryWrapper';
+
+const javascriptExample = `// Apify SDK v3 uses named exports instead of the Apify object.
```

Comment:
This is not really connected to the changes, but I'm not sure about this code example here, it doesn't do any logic. @hanatsai do you remember why we wanted to use this code?

Reply:
Oh yeah, I see now. A big part of the code is missing here.

Please check the code example on the current SDK landing page https://docs.apify.com/sdk/js/